spec/controllers/admin/events_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Admin::EventsController do
+  let!(:conference) { create(:conference) }
+  let!(:organizer_role) { Role.find_by(name: 'organizer', resource: conference) }
+  let!(:organizer) { create(:user, role_ids: organizer_role.id) }
+  let!(:event_without_commercial) { create(:event, program: conference.program) }
+  let!(:event_with_commercial) { create(:event, program: conference.program) }
+  let!(:event_commercial) { create(:event_commercial, commercialable: event_with_commercial, url: 'https://www.youtube.com/watch?v=M9bq_alk-sw') }
+  with_versioning do
+    context 'logged in as an organizer' do
+      before :each do
+        sign_in(organizer)
+      end
+
+      describe 'GET #show' do
+        let(:show_action){ get :show, id: event_without_commercial.id, conference_id: conference.short_title }
```

Comment:
Go with this
https://github.com/openSUSE/osem/blob/master/spec/controllers/admin/roles_controller_spec.rb#L16